backend/users/views_components.py

```diff
@@ -0,0 +1,14 @@
+from .serializers import UserWithSlidesSerializer, UserSerializer
+
+
+class ViewsCommonComponents:
+
+    def check_include_slides_params(query_params):
```

Comment:
this method returns a serializer, I suggest to rename it to a more indicative name.